internal/pkg/template/templates/task/cf.yml

```diff
@@ -39,6 +43,13 @@ Conditions:
     !Not [ !Equals [ !Join [ "", !Ref EntryPoint ], "" ] ]
   HasCustomPlatform:
     !Not [!Equals [!Ref OS, ""]]
+  HasApp:
+    !Not [!Equals [!Ref App, ""]]
+  HasEnv:
+    !Not [!Equals [!Ref Env, ""]]
```

Comment:
Do we really need hasapp and has env?

Reply:
Actually we can remove this and put these conditions directly into `HasAppAndEnv` condition.